image_data_guide/04c_pytorch_training.ipynb

```diff
@@ -0,0 +1,1126 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PyTorch Training with SageMaker (Part 4/4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download | Structure | Preprocessing | **Train Model** "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "* #### Dependencies\n",
+    "* #### Algorithm Hyperparameters\n",
+    "* #### Review the Training Script\n",
+    "* #### Estimator Configuration\n",
+    "* #### Training in Local Mode and Debugging\n",
+    "* #### Training on EC2 Instances\n",
+    "* #### Load Trained Model and Predict"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies\n",
+    "___\n",
+    "For this guide we'll use the SageMaker Python SDK version 2.9.2. By default, SageMaker Notebooks come with version 1.72.0. Other guides provided by Amazon may be set up to work with other versions of the Python SDK so you may wish to roll-back to 1.72.0. We will also be using PyTorch 1.6.0 which can also be rolled back at the end of this guide to 1.4.0."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Update the SageMaker Python SDK and PyTorch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mWARNING: You are using pip version 20.0.2; however, version 20.2.3 is available.\n",
+      "You should consider upgrading via the '/home/ec2-user/anaconda3/envs/pytorch_latest_p36/bin/python -m pip install --upgrade pip' command.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "original_sagemaker_version = !conda list | grep -E \"sagemaker\\s\" | awk '{print $2}'\n",
+    "original_pytorch_version = !conda list | grep -E \"torch\\s\" | awk '{print $2}'\n",
+    "!{sys.executable} -m pip install -q \"sagemaker==2.9.2\" \"torch==1.6.0\" \"torchvision\""
```

Comment:
I tried this without running this line and the notebook worked fine.

Reply:
removed the cell